webapp/portlet/src/main/webapp/vue-apps/common/js/SpaceService.js

```diff
@@ -213,7 +213,7 @@ export function getSpaceNavigations(spaceId) {
 
 export function getSpaces(query, offset, limit, filter, expand) {
   if (!expand) {
-    expand = filter === 'requests' ? 'pending' : limit && 'managers' || '';
+    expand = filter === 'requests' ? 'pending' : limit && 'managers,favorite' || '';
```

Comment:
it's better to specify the `expand` in locations where it's asked instead of changing default value